projects/java-example/build.sh

```diff
@@ -31,7 +31,11 @@ for fuzzer in $(find $SRC -name '*Fuzzer.java' -or -name '*FuzzerNative.java');
   cp $SRC/$fuzzer_basename.class $OUT/
 
   if [[ $fuzzer_basename == *FuzzerNative ]]; then
-    driver=jazzer_driver_asan
+    if [[ $SANITIZER == "address" ]]; then
```

Comment:
I dont like this block, will this need to be done for every java project build.sh. can this be a jazzer_driver wrapper ?

Reply:
You are right, that was a missed opportunity. I have moved the logic into `compile`, which now emits a `jazzer_driver_with_sanitizer` that always has the correct sanitizer baked in.

I have also submitted https://github.com/google/clusterfuzz/pull/2357 to adapt for this change.